migrations/004.do.indexer-stats.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE indexer_query_stats (
+  day DATE NOT NULL PRIMARY KEY,
+  total INT NOT NULL,
+  advertising_http INT NOT NULL
```

Comment:
Perhaps `http_advertised` would be a better column name?

Reply:
I think we can improve this by looking at the topology. We have query stats, and for each day we're tracking..the total number of stats advertising http? (I'm being naive here). We are tracking the total number of tasks and tasks with http advertisement, etc, but the notion of a task is missing.

What do you think about changing the name from `indexer_query_stats.advertising_http` to `task_indexer_stats.advertising_http` or `task_indexing_stats.advertising_http`?

Reply:
We are tracking daily stats of how many tasks (deals tested) have HTTP retrieval advertised in IPNI.

I can rename `_stats` to `_daily_stats`, but it's a long name and we track all stats with daily granularity (e.g. `retrieval_stats`).

I can rename the columns to `tasks_total` and `tasks_advertising_http`, but isn't that too long and a bit redundant?

Reply:
Oh, and we also want to move from "tasks" to "deals" in the near future (see https://github.com/filecoin-station/roadmap/issues/65).

Reply:
That leads me to the following column names:

- `deals_tested`
- `deals_advertising_http`

Reply:
I quite like these two names, I applied the rename in 2ac16e1.

PTAL again

Reply:
89c1762 updates also variable names